operatorcourier/identify.py

```diff
@@ -29,7 +29,7 @@ def get_operator_artifact_type(operatorArtifactString):
                 artifact_type = operatorArtifact["kind"]
                 artifact_name = operatorArtifact['metadata']['name']
             if artifact_type is not None:
-                logger.info('Parsed %s: %s', artifact_type, artifact_name)
```

Comment:
NACK, logging should stay lazy.

I mas set loglevel=ERROR, in first case logger.info() is like noop, in your case a new string is created and then dropped.

Reply:
Sorry - I don't understand what you mean here.